lib/ansible/modules/network/eos/eos_user.py

```diff
@@ -46,6 +46,11 @@
       - The username to be configured on the remote Arista EOS
         device.  This argument accepts a stringv value and is mutually
         exclusive with the C(users) argument.
+  password:
+    description:
+      - The password to be configured on the remote Arista EOS device. The
```

Comment:
Better sounds with `The password to be configured on the remote Arista EOS device to authenticate the user to be created`?